src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow.java

```diff
@@ -199,10 +194,13 @@ public boolean isInTheSameState(double wantedAng) {
     }
 
     public static double getStaticFeedForward(double extenderLength,double elbowAngle) {
-        return (RobotMap.TelescopicArm.Elbow.MIN_Kg + (((RobotMap.TelescopicArm.Elbow.MAX_Kg - RobotMap.TelescopicArm.Elbow.MIN_Kg) * extenderLength)
+        return (RobotMap.TelescopicArm.Elbow.MIN_Kg + (((RobotMap.TelescopicArm.Elbow.MAX_Kg - RobotMap.TelescopicArm.Elbow.MIN_Kg + kS) * extenderLength)
                 / RobotMap.TelescopicArm.Elbow.MAX_KG_MEASUREMENT_LENGTH)) * Math.cos(elbowAngle + RobotMap.TelescopicArm.Elbow.STARTING_ANGLE_RELATIVE_TO_GROUND);
 
     }
+    public static double getDynamicFeedForward(double extenderLength,double elbowAngle) {
+        return getStaticFeedForward(extenderLength, elbowAngle) + kV / Battery.getInstance().getCurrentVoltage();
```

Comment:
kv should be multiplied by speed

Reply:
also my fault but the sparkmax feedforward takes value in volts so dividing by the voltage is wrong

Reply:
and what about falcons? do they also take in voltage?